bin/install

```diff
@@ -60,13 +60,16 @@ get_java() {
 	local base=http://download.oracle.com/otn-pub/java/jdk/
 	local oracle_version=$(echo $version | sed -e 's/\./u/g')
 
+	local s3base=https://s3-eu-west-1.amazonaws.com/repo.proemion.com
+
 	case $version in
 		9.0.1) url=$base$version+11/jdk-$version'_'$(get_current_variant $variant) ;;
 		8.151) url=$base$oracle_version-b12/e758a0de34e24606bca991d704f6dcbf/jdk-$oracle_version-$variant ;;
```

Comment:
@dawn-minion let's upload 151 to s3 and use that here

Reply:
@unicolet Will do

Reply:
@dawn-minion thanks :-) We can leave 131 pointing to s3 since it's already there